frontend/public/components/build.tsx

```diff
@@ -42,6 +43,29 @@ export enum BuildStrategyType {
   Source = 'Source',
 }
 
+const BuildGraphs = requirePrometheus(({build}) => {
+  const podName = _.get(build, ['metadata', 'annotations', 'openshift.io/build.pod-name']);
+  if (!podName) {
+    return null;
+  }
+
+  return <React.Fragment>
+    <div className="row">
+      <div className="col-md-4">
+        <Line title="RAM" query={`pod_name:container_memory_usage_bytes:sum{pod_name='${podName}',container_name='',namespace='${build.metadata.namespace}'}`} />
+      </div>
+      <div className="col-md-4">
+        <Line title="CPU Shares" query={`pod_name:container_cpu_usage:sum{pod_name='${podName}',container_name='',namespace='${build.metadata.namespace}'} * 1000`} />
+      </div>
+      <div className="col-md-4">
+        <Line title="Filesystem (bytes)" query={`pod_name:container_fs_usage_bytes:sum{pod_name='${podName}',container_name='',namespace='${build.metadata.namespace}'}`} />
+      </div>
+    </div>
+
+    <br />
```

Comment:
Nit:  this  makes me wince.  No way to fix with CSS?

Reply:
We could. I was following what was there for pods and nodes. For instance:

https://github.com/openshift/console/blob/master/frontend/public/components/pod.jsx#L151

We'd probably want to fix it in all places.